firestore/testapp/README.md

```diff
@@ -13,7 +13,19 @@ The testapp performs the following:
 -   Gets a pointer to firebase::Auth, and signs in anonymously. This allows the
     testapp to access a Firebase Firestore instance with authentication rules
     enabled.
--   TODO(varconst): describe the Firestore-specific logic
+-   Initializes a Firestore instance, and sets its logging level to
```

Comment:
Ultranit: I think the comma before `and` is extraneous.

Reply:
Done